riscv/tests/riscv_data/alloc.rs

```diff
@@ -3,7 +3,6 @@
 
 use core::alloc::{GlobalAlloc, Layout};
```

Comment:
this file doesn't compile on my system even before I did my changes, because of `extern crate alloc` missing. I can implement an allocator in the mini runtime if need be, or at least a stub so that this PR is merged - and then do the allocator work in a different PR. Like I said, it doesn't compile on my system so I assume it's still work in progress and the fix need not be part of this PR.